opsmngr/opsmngr.go

```diff
@@ -94,6 +96,7 @@ type Client struct {
 	ServerUsage            ServerUsageService
 	ServerUsageReport      ServerUsageReportService
 	LiveMigration          LiveDataMigrationService
+	ServiceVersion         ServiceVersionService
```

Comment:
[q] do you need to duplicate the implementation? if the service is the same we do 
```suggestion
	ServiceVersion         atlas.ServiceVersionService
```